dex-load/src/main/scala/com/wavesplatform/dex/load/ws/WsCollectChangesClient.scala

```diff
@@ -19,16 +19,17 @@ class WsCollectChangesClient(apiUri: String, address: String, aus: String, obs:
 
   private val log = LoggerFactory.getLogger(s"WsApiClient[$address]")
 
-  private val emptyWsAddresState: WsAddressChanges = WsAddressChanges(Address.fromString(address).explicitGet(), Map.empty, Seq.empty, 0L)
-  @volatile private var accountUpdates = emptyWsAddresState
+  private val emptyWsAddressState: WsAddressChanges = WsAddressChanges(Address.fromString(address).explicitGet(), Map.empty, Seq.empty, 0L)
+  @volatile private var accountUpdates = emptyWsAddressState
   private val orderBookUpdates = mutable.AnyRefMap.empty[AssetPair, WsOrderBookChanges]
+  val addressUpdateLeaps = scala.collection.mutable.ArrayBuffer.empty[String]
   @volatile private var gotPings = 0
 
   private val receive: Function[WsServerMessage, Option[WsClientMessage]] = {
     case x: WsPingOrPong => gotPings += 1; x.some
     case x: WsInitial => log.info(s"Connection id: ${x.connectionId}"); none
     case x: WsError => log.error(s"Got error: $x"); throw new RuntimeException(s"Got $x")
-    case diff: WsAddressChanges => accountUpdates = merge(accountUpdates, diff); none
+    case diff: WsAddressChanges => collectLeaps(accountUpdates, diff); accountUpdates = merge(accountUpdates, diff); none
```

Comment:
Could we collect also the total number of `WsAddressChanges` to get the ratio of leaps?

Reply:
fixed